src/Text.Analyzers/Core/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -33,162 +43,465 @@ public abstract class IdentifiersShouldBeSpelledCorrectlyAnalyzer : DiagnosticAn
-                                                                             DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
-                                                                             description: s_localizableDescription,
-                                                                             helpLinkUri: "https://docs.microsoft.com/visualstudio/code-quality/ca1704",
-                                                                             customTags: FxCopWellKnownDiagnosticTags.PortedFxCopRule);
-        internal static DiagnosticDescriptor MethodTypeParameterMoreMeaningfulNameRule = new(RuleId,
-                                                                             s_localizableTitle,
-                                                                             s_localizableMessageMethodTypeParameterMoreMeaningfulName,
-                                                                             DiagnosticCategory.Naming,
-                                                                             DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
-                                                                             description: s_localizableDescription,
-                                                                             helpLinkUri: "https://docs.microsoft.com/visualstudio/code-quality/ca1704",
-                                                                             customTags: FxCopWellKnownDiagnosticTags.PortedFxCopRule);*/
-
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray<DiagnosticDescriptor>.Empty;
-        //ImmutableArray.Create(AssemblyRule, NamespaceRule, TypeRule, MemberRule, MemberParameterRule, DelegateParameterRule, TypeTypeParameterRule, MethodTypeParameterRule, AssemblyMoreMeaningfulNameRule, NamespaceMoreMeaningfulNameRule, TypeMoreMeaningfulNameRule, MemberMoreMeaningfulNameRule, MemberParameterMoreMeaningfulNameRule, DelegateParameterMoreMeaningfulNameRule, TypeTypeParameterMoreMeaningfulNameRule, MethodTypeParameterMoreMeaningfulNameRule);
-
-#pragma warning disable RS1025 // Configure generated code analysis
+        private static readonly SourceTextValueProvider<CodeAnalysisDictionary> _xmlDictionaryProvider = new SourceTextValueProvider<CodeAnalysisDictionary>(ParseXmlDictionary);
+        private static readonly SourceTextValueProvider<CodeAnalysisDictionary> _dicDictionaryProvider = new SourceTextValueProvider<CodeAnalysisDictionary>(ParseDicDictionary);
+        private static readonly CodeAnalysisDictionary _mainDictionary = GetMainDictionary();
+
+        internal static DiagnosticDescriptor FileParseRule = new DiagnosticDescriptor(
+            RuleId,
```

Comment:
You need to add this to the release file (there is a code-fix for this).

Reply:
@pakdev If you switch the descriptor creation to use the helper like below, it will likely fix your build error: https://github.com/dotnet/roslyn-analyzers/blob/edf95f72a20ed1c090a1639c8ec85bb5854fe09e/src/Text.Analyzers/Core/EnumsShouldHavePluralNames.cs#L43-L52

Reply:
Done